config/checkstyle_input_suppressions.xml

```diff
@@ -1292,12 +1292,24 @@
             files="javadoc[\\/]javadoctype[\\/]InputJavadocTypeWhitespace.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableInner.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableInner2.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableNoJavadoc.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableNoJavadoc2.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableNoJavadoc3.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableNoJavadoc4.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]javadocvariable[\\/]InputJavadocVariableNoJavadoc5.java"/>
```

Comment:
@hiron1999 Please do not add suppressions, and try to resolve as many previous suppressions associated with this test.

Reply:
You might have created this branch directly from your last commited branch, so this branch has been cloned similar to previous one.

Reply:
Resolved.

Reply:
@hiron1999 Please resolve this one. Also, Please comment `Done` to each suggested changes so as to keep track which ones have been done.